containers/python/python_container_entry.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+
+conda env create -f=/model/environment.yml -n container-env
+source activate container-env
+python -u /container/python_container.py
```

Comment:
So on further discussion, we don't want to run Python in unbuffered mode. Remove the `"-u"`. Instead, let's flush the buffered output at the end of each iteration of message processing.

Add the following 2 lines on [line 175 of`rpc.py`](https://github.com/ucbrise/clipper/blob/develop/containers/python/rpc.py#L175)

```py
sys.stdout.flush()
sys.stderr.flush()
```

Make sure the flushes are outside the `else` statement like this:

```py
            else:
                received_msg = Message(msg_id_bytes, [])
                response = self.handle_feedback_request(received_msg)
                response.send(self.socket)
                print("recv: %f us" % ((t2 - t1).microseconds))
            sys.stdout.flush()
            sys.stderr.flush()

```